test/team_test.rb

```diff
@@ -0,0 +1,75 @@
+require './lib/player'
+require './lib/team'
+require 'minitest/autorun'
+require 'minitest/pride'
+
+class TeamTest < MiniTest::Test
+
+  def test_team_exists
+    france = Team.new("France")
+
+    assert_instance_of Team, france
+  end
+
+  def test_it_has_a_country
+    france = Team.new("France")
+    expected = "France"
+    actual = france.country
+
+    assert_equal expected, actual
+  end
+
+  def test_france_starts_with_empty_team
```

Comment:
Don't say "france_starts_with", because this is a test for all Teams, not just france